emulators/qemu/Portfile

```diff
@@ -52,23 +52,15 @@ patchfiles-append       patch-qemu-configure.diff
 # "error: unknown type name 'fpunchhole_t'" pre-10.12.4
 patchfiles-append       patch-qemu-punchhole.diff
 
-# "cp: illegal option -- a"
-patchfiles-append       patch-qemu-scripts-entitlements.diff
-
```

Comment:
FWIW, this was applied upstream. https://github.com/qemu/qemu/commit/4006a27c5e44734350009262efb0e2ec8da5ff09